docs/src/customization.md

```diff
@@ -37,7 +37,7 @@ If you want to bind some action to a mouse, open `Preferences / Browser Packages
 | Linux    | `/User/Default (Linux).sublime-mousemap` |
 | Mac      | `/User/Default (OSX).sublime-mousemap` |
 
-Here is an example mouse binding that triggers LSP's "go to symbol definition" command on pressing the <kbd>ctrl</kbd>+<kbd>left click</kbd>:
+Here is an example of mouse binding that triggers LSP's "go to symbol definition" command on pressing the <kbd>ctrl</kbd>+<kbd>left click</kbd>:
```

Comment:
English is not my first language, but "example of mouse binding" sounds really weird to me. Shouldn't it be "example of a mouse binding", or alternatively leave it as it was before?